lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.spec.ts

```diff
@@ -116,7 +134,7 @@ describe('ContentMetadataComponent', () => {
      * @returns native element
      */
     function getAssignCategoriesBtn(): HTMLButtonElement {
```

Comment:
If you're targeting the `adf-metadata-categories-title` class the name of the function is not really correct anymore